code_generator/End2End/GeneratedDependencies/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
I think you can delete this file as we won't be distributing the built out `dll`

Reply:
Done